### ISSUE TITLE
source R script names to represent function calls

### ISSUE THREAD

**Issue by kgaonkar6** [d3b-center/annoFuse#13]
# Description 

General issue:
R score file names and functions don't match up

Specifics:
`plotRecurrentFusion` is in `plot_recurrent_fusion.R`
`plotRecurrentGenes` is in `plot_recurrent_genes.R`
`plotBreakpoints` is in 'plot_breakpoints.R`
`fusion_multifused` is in `fusionMultifused.R`
`annotate_fusion_calls` is in `annotate_fusion.R`
`expressionFilterFusion` is in `expression_filtering.R`

Styler for all scripts 

All functions to be in snake_case as per discussion below

How will work on this
@federicomarini  will work on changing the names in the code
@kgaonkar6  to update the Fig1 function names

**Comment by federicomarini** [comment]
If you want I can just take care of this in the existing PR @kgaonkar6

**Comment by federicomarini** [comment]
Oh, while we are at it: shall we set for "one style", i.e. `camelCase` or `snake_case`? I'm slightly preferring the latter.

Only downside: we'd need to update also Fig. 1 where the schematics include the names of the function calls
Plus: well, we'd gain in consistency, and code looks tidier.

EDIT: `camelCase` is what we use in some column names of the outputs, so yes, probably a bonus point for having functions as `snake_case`

**Comment by kgaonkar6** [comment]
I agree snake_case would work best for functions.
 
If you can add to the existing PR that would be great, thanks! I can work on updating the Fig 1 for the respective changes.

**Comment by federicomarini** [comment]
Sure, will do!

**Comment by federicomarini** [comment]
Done in https://github.com/federicomarini/annoFuse/tree/func_naming

Please see the namespace file for an overview of the newer names, if it is fine I'll merge into master on my fork.
https://github.com/federicomarini/annoFuse/blob/func_naming/NAMESPACE

**Comment by kgaonkar6** [comment]
Thanks! The names look right to me in the branch and the namespace file.

**Comment by federicomarini** [comment]
Okiedokie, merging in in the main PR then!

**Comment by kgaonkar6** [comment]
Closing since @federicomarini fixed this in #11

**Comment by federicomarini** [comment]
To be done yet - 
report_fuse and shiny_fuse will become reportFuse and shinyFuse, in analogy with the package name

**Comment by federicomarini** [comment]
Done once #48 is merged in!
